python/arcticdb/version_store/_store.py

```diff
@@ -447,6 +447,7 @@ def write(
         metadata: Optional[Any] = None,
         prune_previous_version: Optional[bool] = None,
         pickle_on_failure: Optional[bool] = None,
+        ensure_sorted: bool = False,
```

Comment:
Is `ensure_sorted` the best name for this? If `is_monotonic_decreasing` is `True` on the input data, it's definitely sorted, but we will reject anything that isn't monotonically _increasing_

Reply:
Correct, but Arcticc point of view does not support currently the decreasing approach. So, as its eyes, it is not sorted. But I will leave it open to discussion for next week

Reply:
validate_index_ordering

Reply:
validate_index